docs/framework/guides/contributing/development-environment.md

```diff
@@ -8,13 +8,17 @@ order: 10
 The CKEditor 5 codebase is divided into multiple [npm](http://npmjs.com/) packages, each developed in a separate Git repository. The main package is [`ckeditor5`](https://github.com/ckeditor/ckeditor5) which installs all project dependencies and various development-related resources such as:
```

Comment:
"each developed in a separate Git repository"

Reply:
Addressed in c2a07ff.